src/index.html

```diff
@@ -2,16 +2,160 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input type="text" name="surname">
+        </label>
```

Comment:
Follow the requirements, please
<img width="355" alt="image" src="https://user-images.githubusercontent.com/99403198/216760940-0841c71a-46fb-4fc3-af4e-6d1ad71c9a7a.png">

Reply:
ok. I will put it for an entire form, if it's ok